File: miracle-extra/src/main/java/com/miracle/extra/domain/ErpInfo.java
package com.miracle.extra.domain;

import com.miracle.common.core.domain.BaseEntity;
import lombok.Data;

@Data
public class ErpInfo extends BaseEntity {
    private String workerName;
    private String workerCode;
    private String workshopName;
    private String workshopCode;
    private String materialName;
    private String materialCode;
}

File: miracle-extra/src/main/java/com/miracle/extra/domain/ErpInfo.kt
package com.miracle.extra.domain;

import com.miracle.common.core.domain.BaseEntity

data class ErpInfo (
    var workerName: String? = null,
    var workerCode: String? = null,
    var workshopName: String? = null,
    var workshopCode: String? = null,
    var materialName: String? = null,
    var materialCode: String? = null,
) : BaseEntity()
